### PR TITLE
VEP2 - change to release cadence and release numbering

### LINKED ISSUE #2: Major release every 3 months instead of every 12 weeks
We currently do a major release every 12 weeks per VEP-1(#1). I would like to change this to every 3 months / 13 weeks so that the releases are at a more or less fixed time every year.

If we adopt this proposal, 7.0 will be released on July 28, 2020 instead of July 21, 2020.

EDIT:
Another change proposed by @morgo:
Instead of making patch releases monotonic as described [here](https://github.com/vitessio/enhancements/blob/master/veps/vep-1.md#versioning-and-release-cadence), we will modify the versioning to follow SemVer. This is because we have not actually been making weekly releases, we are doing only major releases + on-demand patch releases.
EDIT 2:
Clarify that SemVer is for patch releases. Major version number will change every 3 months.

### PR DESCRIPTION
**Pull request by deepthi**
Fixes #2 
Signed-off-by: deepthi <deepthi@planetscale.com>

### CODE REVIEW

File: veps/vep-2.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+## Abstract
+
+VEP-1(#1) established a release cycle for Vitess. Now that there have been 3 major releases, this VEP proposes some changes to the release cycle based on that experience.
+All other sections of VEP-1 will continue to be in force without any change.
+
+## Versioning and Release Cadence
+
+The proposed release cycle is best described as _Monotonic Versioning_ with a release cadence similar to Kubernetes. The terminology `COMPATIBILITY.RELEASE` used in this document is directly sourced from the [Monotonic Versioning Manifesto](http://blog.appliedcompscilab.com/monotonic_versioning_manifesto/). Monotonic versioning itself is a modified version of [Semantic Versioning](https://semver.org/).
```
Comment (deepthi) [inline veps/vep-2.md]: @morgo I could use some help on getting this language right. If what we are proposing is no longer monotonic, I'll need to rewrite this section.

Comment (derekperkins) [reply veps/vep-2.md]: From my reading of that link, we're not actually doing monotonic versioning either. I'd just leave the doc as is and change the week count

Comment (deepthi) [reply veps/vep-2.md]: I still had to address the patch release numbering. Right now we are doing releases like 6.0.YY-yyyymmdd. We want to do 6.0.1, 6.0.2 etc.

File: veps/vep-2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## Versioning and Release Cadence
+
+1. The `COMPATIBILITY` number will be incremented every 13 weeks. For example, the next major release will have a `COMPATIBILITY` number of `7.0`.
+2. Each `COMPATIBILITY` number will be supported by the Open Source Vitess community for 9 months after the initial release. _Support Lifecycle_ is described in VEP-1.
+3. Patch releases will be created as needed. Patch releases will follow SemVer. The numbering scheme will be `COMPATIBILITY.PATCH_NUMBER`. For example, the first patch release on `7.0`, if there is one, will be `7.0.1`. `PATCH_NUMBER` will be monotonically increasing.
+4. Events that trigger patch releases are major regressions in functionality or performance, and CVEs.
+
+## Release Criteria
+
+In addition to the criteria established in VEP-1, a nightly performance benchmark will be established. Any changes that impact performance by more than 5% as measured against the most recent major release will need to be addressed before another release can be made.
```
Comment (deepthi) [inline veps/vep-2.md:26]: @dkhenry Is this what you had in mind? I threw 5% up as a strawman, we can debate what that threshold is.

**Review by dweitzman — APPROVED** [review]

**Review by rafael — APPROVED** [review]
LGTM

**Pull request merged**